libkbfs/block_tree.go

```diff
@@ -609,6 +610,45 @@ func (bt *blockTree) setParentOffsets(
 	return newDirtyPtrs, newUnrefs, nil
 }
 
+func (bt *blockTree) String() string {
+	block, _, err := bt.getter(
+		nil, bt.kmd, bt.rootBlockPointer(), bt.file, blockRead)
+	if err != nil {
+		return "ERROR: " + err.Error()
+	}
+
+	level := []BlockWithPtrs{block}
+	res := "\n---------------\n"
```

Comment:
Perhaps use a `bytes.Buffer` here to reduce allocations? It doesn't seem like this will be used much though, so not a big deal.

Reply:
Yeah, it's completely unused for now, I just wanted it here for whenever I'm debugging something tricky.  I'll put in a TODO to make it better if ever used in production.